ckan/pastertemplates/template/setup.py_tmpl

```diff
@@ -54,6 +54,7 @@ setup(
     # You can just specify the packages manually here if your project is
     # simple. Or you can use find_packages().
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
+    namespace_packages=['ckanext', 'ckanext.{{ project_shortname }}'],
```

Comment:
@wardi: I was unsure about that, too. I checked some existing CKAN extensions and in most cases both `ckanext` and the extension's sub-package were declared as namespace packages. For example, [`ckanext.harvest`](https://github.com/ckan/ckanext-harvest/blob/master/setup.py#L19), [`ckanext.geoview`](https://github.com/ckan/ckanext-geoview/blob/master/setup.py#L17), and [`ckanext.dcat`](https://github.com/ckan/ckanext-dcat/blob/master/setup.py#L18) are namespace packages. Of course there are also counter-examples (e.g. [`ckanext.pdfview`](https://github.com/ckan/ckanext-pdfview/blob/master/setup.py#L56)). So I decided to use the more popular approach.

The benefit of registering `ckanext.my_extension` as a namespace package would be that one could distribute sub-extensions for `my_extension`. While I don't think that this is a common use case I also don't see any disadvantages of declaring `ckanext.my_extension` as a namespace package.

In the end I think both approaches are valid and I'd be happy to change it according to your preferences.